File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task182_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <regex>
#include <string>

std::string cleanPhoneNumber(std::string phoneNumber) {
    std::regex re("[^\\d]");
    phoneNumber = std::regex_replace(phoneNumber, re, "");
    if (phoneNumber.size() > 0 && phoneNumber[0] == '1') {
        phoneNumber.erase(0, 1);
    }
    return phoneNumber;
}

int main() {
    std::string phoneNumber;
    std::cout << "Enter phone number: ";
    std::getline(std::cin, phoneNumber);
    std::cout << cleanPhoneNumber(phoneNumber) << std::endl;
    return 0;
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task182_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.Scanner;
import java.util.regex.Matcher;
import java.util.regex.Pattern;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter phone number: ");
        String phoneNumber = scanner.nextLine();
        System.out.println(cleanPhoneNumber(phoneNumber));
    }

    public static String cleanPhoneNumber(String phoneNumber) {
        String cleanNumber = phoneNumber.replaceAll("[^\\d]", "");
        if (cleanNumber.startsWith("1")) {
            cleanNumber = cleanNumber.substring(1);
        }
        return cleanNumber;
    }
}